src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,18 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
-);
+import { TodoInfo } from '../TodoInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[]
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map(todo => {
+        return (
+          <TodoInfo todo={todo} key={todo.id} />
+        );
+      })}
```

Comment:
if you don't have any additional action with an element which you mapping, then don need to write return:
```suggestion
      {todos.map(todo => (
        <TodoInfo todo={todo} key={todo.id} />
      ))}
```